### PR TITLE
Consolidate Notice Colors

### PR DESCRIPTION
**Pull request by carkom**
This is a proof of concept for consolidating colors to make the apps color scheme more consistent. We are using 5 different shades of dark green for text fields. Looking to slim this down to 3 and consolidate background colors a bit as well and clean up the naming standards.

It should also help to make the palette more compact allowing for easier creation of more themes.

There are places where the color hasn't actually changed but the name of the color has. I'll try to point out the bits where the color has changed by one shade so you don't have to review all the ones where color is the same but name is different.

### CODE REVIEW

File: packages/desktop-client/src/components/accounts/Balance.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ export function Balances({
               value < 0
                 ? theme.errorText
                 : value > 0
-                ? theme.noticeText
+                ? theme.noticeTextLight
                 : theme.pageTextSubdued,
```
Comment (carkom) [inline packages/desktop-client/src/components/accounts/Balance.js:163]: No change

File: packages/desktop-client/src/components/accounts/MobileAccounts.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ function AccountCard({ account, updated, getBalanceQuery, onSelect }) {
             {account.bankId && (
               <View
                 style={{
-                  backgroundColor: theme.noticeText,
+                  backgroundColor: theme.noticeBackgroundDark,
                   marginLeft: '-23px',
```
Comment (carkom) [inline packages/desktop-client/src/components/accounts/MobileAccounts.js:104]: No change

File: packages/desktop-client/src/components/accounts/Reconcile.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export function ReconcilingMessage({
         {targetDiff === 0 ? (
           <View
             style={{
-              color: theme.noticeText,
+              color: theme.noticeTextLight,
               flex: 1,
```
Comment (carkom) [inline packages/desktop-client/src/components/accounts/Reconcile.js:49]: No change

File: packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx [hunk 4]
```diff
@@ -93,7 +93,7 @@ function CategoryList({
                   alignItems: 'center',
                   fontSize: 11,
                   fontWeight: 500,
-                  color: theme.noticeAccent,
+                  color: theme.noticeTextMenu,
                   padding: '6px 8px',
```
Comment (carkom) [inline packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx:97]: No change

File: packages/desktop-client/src/components/autocomplete/PayeeAutocomplete.js [hunk 5]
```diff
@@ -99,7 +99,7 @@ function PayeeList({
             <View
               style={{
                 display: 'block',
-                color: theme.noticeAccent,
+                color: theme.noticeTextMenu,
                 borderRadius: 4,
```
Comment (carkom) [inline packages/desktop-client/src/components/autocomplete/PayeeAutocomplete.js:103]: No change

File: packages/desktop-client/src/components/budget/report/BudgetSummary.tsx [hunk 6]
```diff
@@ -99,7 +99,7 @@ function IncomeProgress({ current, target }: IncomeProgressProps) {
   return (
     <PieProgress
       progress={frac}
-      color={over ? theme.errorText : theme.noticeText}
+      color={over ? theme.errorText : theme.noticeTextLight}
       backgroundColor={over ? theme.errorBackground : theme.pageBackground}
```
Comment (carkom) [inline packages/desktop-client/src/components/budget/report/BudgetSummary.tsx:103]: No change

File: packages/desktop-client/src/components/budget/report/BudgetSummary.tsx [hunk 7]
```diff
@@ -132,7 +132,7 @@ function ExpenseProgress({ current, target }: ExpenseProgressProps) {
...
```
Comment (carkom) [inline packages/desktop-client/src/components/budget/report/BudgetSummary.tsx:136]: No change

File: packages/desktop-client/src/components/manager/subscribe/ChangePassword.tsx [hunk 8]
```diff
@@ -73,7 +73,7 @@ export default function ChangePassword() {
         <Text
           style={{
             marginTop: 20,
-            color: theme.noticeText,
+            color: theme.noticeTextLight,
             borderRadius: 4,
```
Comment (carkom) [inline packages/desktop-client/src/components/manager/subscribe/ChangePassword.tsx:77]: No change

File: packages/desktop-client/src/components/reports/Change.js [hunk 9]
```diff
@@ -10,7 +10,7 @@ function Change({ amount }) {
     <Block
       style={{
         ...styles.smallText,
-        color: amount < 0 ? theme.errorText : theme.noticeText,
+        color: amount < 0 ? theme.errorText : theme.noticeTextLight,
       }}
```
Comment (carkom) [inline packages/desktop-client/src/components/reports/Change.js:14]: No change

File: packages/desktop-client/src/components/schedules/SchedulesTable.tsx [hunk 10]
```diff
@@ -168,7 +168,7 @@ export function ScheduleAmountCell({
       <Text
         style={{
           flex: 1,
-          color: num > 0 ? theme.noticeText : theme.tableText,
+          color: num > 0 ? theme.noticeTextLight : theme.tableText,
           whiteSpace: 'nowrap',
```
Comment (carkom) [inline packages/desktop-client/src/components/schedules/SchedulesTable.tsx:172]: No change

File: packages/desktop-client/src/components/settings/Encryption.tsx [hunk 11]
```diff
@@ -26,7 +26,7 @@ export default function EncryptionSettings() {
       primaryAction={<Button onClick={onChangeKey}>Generate new key</Button>}
     >
       <Text>
-        <Text style={{ color: theme.noticeText, fontWeight: 600 }}>
+        <Text style={{ color: theme.noticeTextLight, fontWeight: 600 }}>
           End-to-end Encryption is turned on.
```
Comment (carkom) [inline packages/desktop-client/src/components/settings/Encryption.tsx:30]: No change

File: packages/desktop-client/src/components/transactions/MobileTransaction.js [hunk 12]
```diff
@@ -1001,7 +1001,9 @@ class Transaction extends PureComponent {
                   style={{
                     width: 11,
                     height: 11,
-                    color: cleared ? theme.noticeText : theme.altButtonBareText,
```
Comment (carkom) [inline packages/desktop-client/src/components/transactions/MobileTransaction.js:1004]: No change

File: packages/desktop-client/src/components/transactions/TransactionsTable.js [hunk 13]
```diff
@@ -416,7 +416,7 @@ function StatusCell({
 
   let statusColor =
     status === 'cleared'
-      ? theme.noticeText
```
Comment (carkom) [inline packages/desktop-client/src/components/transactions/TransactionsTable.js:419]: No change

**Review by MatissJanis — APPROVED** [review]
LGTM on my end.

Is this just a POC? I'm happy to merge it unless you plan to continue working in this PR.

**Comment by carkom** [comment]
No, not just POC. It's more like the first of many 🤣

**Pull request merged**